tests/integration/components/g-map/info-window-test.js

```diff
@@ -111,11 +111,11 @@ module('Integration | Component | g-map/info-window', function (hooks) {
   test('it attaches an info window to a marker', async function (assert) {
     await render(hbs`
       <GMap @lat={{this.lat}} @lng={{this.lng}} @zoom={{6}} as |g|>
-        <g.marker @lat={{55}} @lng={{2}} as |m|>
+        <g.advancedMarker @lat={{55}} @lng={{2}} as |m|>
```

Comment:
@sandydoo 👀 here:
For some strange reason if `infoWindow` is used with a `marker`, the `getPosition()` doesn't work and hence the `isVisible()` doesn't work. Hence I've used `advancedMarker` instead.

Reply:
You were right about this:
> Looks like a test race condition. Lets try to keep the original marker tests.

And your suggestion to add the `waitUntil` worked 💯 

Regarding:
> Lets also add tests for the advanced marker here as well. You can just copy-paste the marker ones.

I've already added both tests as well as docs for `advanced-marker`